ee/modules/110-istio/hooks/migration_get_rid_of_1x10x1_heritage.go

```diff
@@ -19,5 +19,6 @@ var _ = sdk.RegisterFunc(&go_hook.HookConfig{
 
 func removeKalaiCRD(input *go_hook.HookInput) error {
 	input.PatchCollector.Delete("apiextensions.k8s.io/v1", "CustomResourceDefinition", "", "monitoringdashboards.monitoring.kiali.io", object_patch.InForeground())
+	input.PatchCollector.Delete("admissionregistration.k8s.io/v1", "ValidatingWebhookConfiguration", "", "istiod-d8-istio", object_patch.InForeground())
```

Comment:
The function is named `removeKalaiCRD`.
1. I believe this is `Kiali`.
2. Now it also removes the webhook, so the name does not match what this function is doing anymore.

Reply:
Yes, indeed, I hadn't noticed that point. Thanks!